oak-blob-cloud/src/main/java/org/apache/jackrabbit/oak/blob/cloud/s3/S3Backend.java

```diff
@@ -1368,8 +1370,15 @@ public KeyRenameThread(String oldKey) {
     /**
      * Enum to indicate remote storage mode
      */
-    private enum RemoteStorageMode {
+    enum RemoteStorageMode {
         S3,
-        GCP
+        GCP;
```

Comment:
```suggestion
                enum RemoteStorageMode {
        S3("s3"),
        GCP("gcp");

        RemoteStorageMode(String name) {
        }

        @NotNull
        static RemoteStorageMode fromString(String mode) {

            switch (mode.toLowerCase()) {
                case "s3": return S3;
                case "gcp": return GCP;
                default: return S3;
            }
        }
```

Reply:
modified the code using this functionality, so now this is not required.